src/app/component/auth/new-sign-up/new-sign-up.component.html

```diff
@@ -1 +1,106 @@
-<p>new-sign-up works!</p>
+<div class="main-container">
+  <div class="main-image">
+    <img [src]="signUpImgs.mainImage"
+         alt="main-image"
+    >
+  </div>
+
+  <div class="main-content">
+    <div class="close-btn">
+      <div class="close-btn-img">
+        <a (click)="closeSignUpWindow()">
+          <img [src]="signUpImgs.cross"
+               alt="cross"
+               class="cross-img"
+          >
+        </a>
+      </div>
+    </div>
+    <div class="main-content-container">
+      <div class="title">
+        <h1 class=""title-text>
+          Вітаємо!
+        </h1>
+      </div>
+      <div class="subtitle">
+        <h2 class="subtitle-text">
+          Будь ласка, внесіть свої дані для реєстрації.
+        </h2>
+      </div>
+      <form action="submit">
+        <div class="form-content-container">
+          <label class="content-label">
+            Електронна пошта
+          </label>
+          <input type="email"
+                 class="main-data-input"
+                 placeholder="example@email.com"
+          >
+          <label class="content-label">
+            Ваше ім’я
```

Comment:
Please change the color according to the mockup. Placeholder should be gray, populated input black(#494A49)